packages/grid/_modules_/grid/hooks/features/core/useGridState.ts

```diff
@@ -1,22 +1,21 @@
 import * as React from 'react';
 import { ApiRef } from '../../../models/api/apiRef';
-import { useRafUpdate } from '../../utils/useRafUpdate';
 import { GridState } from './gridState';
 import { useGridApi } from './useGridApi';
 
 export const useGridState = (
   apiRef: ApiRef,
 ): [GridState, (stateUpdaterFn: (oldState: GridState) => GridState) => void, () => void] => {
   const api = useGridApi(apiRef);
-  const [, forceUpdate] = React.useState();
-  const [rafUpdate] = useRafUpdate(apiRef, () => forceUpdate((p: any) => !p));
-
+  const forceUpdate = React.useCallback(
+    () => apiRef.current.forceUpdate(() => apiRef.current.state),
```

Comment:
```suggestion
    () => apiRef.current.forceUpdate(),
```

Reply:
No I want to pass a func so when I call it will rerender everything

Reply:
This is equivalent to

```suggestion
    () => apiRef.current.forceUpdate({}),
```